### DISCUSSION TITLE
Limit amount of characters

### DISCUSSION THREAD

**Discussion by itaysharir** [FelixKratz/SketchyBar#251] [Q&A] [answered]
![Screen Shot 2022-09-17 at 19 37 39](https://user-images.githubusercontent.com/87126382/190867280-62d22896-c65a-421d-9696-fc8ae61235e0.jpg)
How can i prevent this from happening?

btw I know i'm asking a lot of questions, This is probably the last one. Sorry to bother you again.

**Comment by FelixKratz** [comment, marked as answer]
When you get the title of the window from yabai you can simply cut it with e.g. sed:
```bash
TITLE=$(yabai -m query --windows --window | jq -r '.title' | sed 's/\(.\{30\}\).*/\1.../'
sketchybar --set $NAME label="$TITLE"
```
This example cuts the title after 30 characters and adds "..." to the end if the string was truncated, titles under 30 characters do not get changed.

**Comment by itaysharir** [comment]
Thanks!
